client/settings/payment-request-section/styles.scss

```diff
@@ -0,0 +1,69 @@
+@import '../../styles/abstracts/styles.scss';
+
+.express-checkouts {
+	.express-checkouts-list {
+		.express-checkout {
+			display: flex;
+			margin: 0;
+			padding: 16px 24px 14px 24px;
+			background: #fff;
+			flex-wrap: wrap;
+			justify-content: flex-start;
+			align-items: center;
+
+			@include breakpoint( '>660px' ) {
+				padding: 24px 24px 24px 24px;
+				flex-wrap: nowrap;
+			}
+
+			&:not( :last-child ) {
+				box-shadow: inset 0 -1px 0 #e8eaeb;
+			}
+
+			&__checkbox {
+				display: flex;
+				align-items: center;
+				justify-content: center;
+
+				.components-base-control__field {
+					margin: 0 4px 0 0;
+				}
+			}
+
+			&__icon {
+				border-radius: 2px;
+				display: none;
+				flex: 0 0 63.69px;
+				height: 40px;
+				margin: 1px 17px 1px 1px; // 1px to accommodate for box-shadow
+				box-shadow: 0 0 0 1px #ddd;
+
+				@include breakpoint( '>660px' ) {
+					display: flex;
+				}
+			}
+
+			&__label {
+				display: inline-block;
+				font-size: 14px;
+				font-weight: 600;
+				line-height: 20px;
+				color: $gray-900;
+				margin-bottom: 4px;
+			}
+
+			&__link {
+				margin-left: auto;
+				padding-left: 16px;
+
+				a {
+					white-space: nowrap;
+				}
+			}
+
+			.components-checkbox-control__label {
+				display: none;
+			}
```

Comment:
This section looks a bit boxed up on mobile.

If we add this to the label container, it looks more like the Payment methods section:

```css
&__label-container {
	flex: 1;
}
```

| Before | After |
| -- | -- |
| <img width="400" alt="Express checkouts section on mobile CSS Before" src="https://user-images.githubusercontent.com/5509901/198415150-b74073be-84ab-49bd-89c5-77dcef59dff9.png"> | <img width="400" alt="Express checkouts section on mobile CSS After" src="https://user-images.githubusercontent.com/5509901/198415157-81d5f2f1-b6e2-476d-aa90-2ddabb60733f.png"> |

Reply:
Good catch, updated!